javalib/src/main/scala/java/util/AbstractList.scala

```diff
@@ -1,3 +1,244 @@
 package java.util
 
-abstract class AbstractList[E] extends AbstractCollection[E] with List[E]
+import java.util.{Iterator => JIterator}
```

Comment:
We are inside `java.util`, so I would leave out this import, and just use `Iterator` in the code below.